lib/photo_exporter.rb

```diff
@@ -0,0 +1,46 @@
+class PhotoExporter
+  attr_reader :user
+
+  def initialize user
+    @user = user
+  end
+
+  def perform_export_photos!
+    temp_zip = Tempfile.new([user.username, '_photos.zip'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.